src/Uno.UI/UI/Xaml/IFrameworkElement.cs

```diff
@@ -320,17 +320,19 @@ public static CGSize SizeThatFits(IFrameworkElement e, CGSize size)
 				return new CGSize(0, 0);
 			}
 
-			var width = e.Width
-				.NumberOrDefault(size.Width)
+			var (min, max) = e.GetMinMax();
+
+			var width = size
+				.Width
 				.NumberOrDefault(nfloat.PositiveInfinity)
-				.LocalMin(e.MaxWidth.NumberOrDefault(nfloat.PositiveInfinity))
-				.LocalMax(e.MinWidth.NumberOrDefault(nfloat.NegativeInfinity));
+				.LocalMin((nfloat)max.Width)
```

Comment:
We're not using MaxWidth anymore ?

Reply:
It's done elsewhere in the layout process.

Reply:
The "min" is removed, not the "max" ;-)